picard/util/imageinfo.py

```diff
@@ -17,8 +17,8 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-import StringIO
 import struct
+from picard.compat import StringIO
```

Comment:
If we're moving to Python 3, just remove `compat.py` and use whatever the correct import is.